### ISSUE TITLE
 [aria-*] attributes do not match their roles + <frame> or <iframe> elements do not have a title

### ISSUE THREAD

**Issue by mitko-janeski** [GoogleChrome/lighthouse#13667]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://www.arena-top100.com/

### What happened?

Google Adsense Ads generate these problems
[aria-*] attributes do not match their roles + <frame> or <iframe> elements do not have a title

### What did you expect?

Google Ads and their third party services should be ignored on reports since there is no way for us to fix that problem if so how can we fix this ?

### How were you running Lighthouse?

PageSpeed Insights, Chrome DevTools

### Lighthouse Version

Version 100.0.0.0

**Comment by adamraine** [comment]
Lighthouse cannot determine whether or not some content is loaded by a third-party service. At the end of the day, the page content is failing our accessibility checks. I would suggest you file an issue with Google Adsense as the problem appears to be on their end.

**Comment by connorjclark** [comment]
We provide a toggle in the report to narrow the provided suggestions to what would be most actionable. (however, we currently don't support this for the a11y category, see #13668)

The Lighthouse report is meant to be an objective analysis of the page, and so we don't wish to ignore components of it. At the end of the day, the web author determines what is on the page, and Lighthouse will review the result of that.

**Comment by paulirish** [comment]
Related axe issue regarding why `<ins>` elements cant have an `aria-label` attribute: https://github.com/dequelabs/axe-core/pull/2935

**Comment by ennkh** [comment]
me too, I have the same problem on my website https://www.computer-pdf.com/

**Comment by soufianesabiri** [comment]
Same issue with my site https://www.easytutoriel.com/
Any news on this? I fixed everything and this is the only thing that remains...

**Comment by halilesen** [comment]
Yes. Mee to. If we remove and add them with JS, will it be a policy violation?

**Comment by adamraine** [comment]
This issue has been resolved for a while, making page elements filterable by third party is covered by https://github.com/GoogleChrome/lighthouse/issues/13668.
